### ISSUE TITLE
What's the meaning of output file of type .pkl?

### ISSUE THREAD

**Issue by hiahiahia123456** [open-mmlab/mmaction#88]
Hello. I'd like to use the results from the testing step for further research.  But I did not find the meaning of the result.pkl generated by the test.py. It is the softmax metrics or something else?

**Comment by kennymckormick** [comment]
It depends on whether u use the --use_softmax option. With this option on, the output is the softmax score (sum to 1), otherwise it's the logits of 400 classes.
